client/components/happychat/style.scss

```diff
@@ -52,6 +52,10 @@
 	right: 12px;
 	font-size: 14px;
 	display: none;		/* disabled */
+	box-sizing: border-box;
+
+	transform: translateZ(0);
```

Comment:
I'm not sure, that's a good question! The only "older" browser would be IE, which doesn't support will-change at all, and I'm unsure if the backface visibility and translate tricks perform the same function on IE or Edge...